install/container/tipi-rust.sh

```diff
@@ -98,7 +98,7 @@ mkdir -p ${CARGO_HOME}
 mkdir -p ${RUSTUP_HOME}
 
 echo -e "Installing rust ${RUSTUP_TOOLCHAIN} using rustup from https://sh.rustup.rs"
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --no-modify-path --default-toolchain ${RUSTUP_TOOLCHAIN} -y
+(curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs || echo "echo 'Failed to download rustup - aborting; exit 1") | sh -s -- --no-modify-path --default-toolchain ${RUSTUP_TOOLCHAIN} ${cli_arg_yes}
```

Comment:
If the file doesn't contain proper content, like empty, I think you will still end up with no error as piping to sh will just accept anything. I would download to tmp and run that.

Reply:
I even believe to have seen invalid syntax errors not end up with exit 1 when used with pipe. But it feels so weird that I feel it needs a recheck.